src/screens/HelpScreen.tsx

```diff
@@ -1,13 +1,152 @@
 import React from 'react';
-import { Text } from 'react-native';
+import { Image, Linking, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
 import MasterContainer from '../components/generic/MasterContainer';
+import FONTS from '../assets/fonts';
+import COLORS from '../assets/colors';
+import AS from '../assets/accessibilityStrings';
```

Comment:
Misschien hier `ACCESSIBILITY_STRINGS` van maken, is iets duidelijker dan AS

Reply:
Behalve dan het feit dat ik iedere kans aangrijp om 'Ass' te schrijven of iets in de buurt daarvan, vind ik ACCESSIBILITY_STRINGS net te lang. ACC_STRS ook goed?

Reply:
Ja `ACC_STRS` lijkt me ook goed 👍